docs/notebooks/how_bambi_works.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>later \pmb{u} can be easily confused with \mu, maybe use a letter that stand out more clearly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I'm still unsure about the change. I understand the possible confusion, but I don't want invent a new notation. \pmb{u} is usually used in mixed models literature and so I used it./ Other letter I can think of is \pmb{b}... I'll keep thinking about it</p>